src/lib/pickles/plonk_checks/scalars.ml

```diff
@@ -252,6 +318,324 @@ module Tick : S = struct
               * cell (var (Witness 3, Curr))
               * cell (var (Witness 4, Curr))
             + cell (var (Coefficient 9, Curr)) ) )
+    + if_feature
+        ( LookupTables
+        , (fun () ->
+            alpha_pow 24
+            * ( vanishes_on_last_4_rows
+              * ( cell (var (LookupAggreg, Next))
+                  * ( if_feature
+                        ( LookupsPerRow 0
```

Comment:
So this means, if the lookups per row is zero then uses the multiplicand of the gamma * beta + ..... all of that, otherwise it multiplies by one (so remains unchanged), right? I am asking because I get a bit lost due to indentation -_-'